priv/repo/migrations/20221221100911_alter_organizers.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Bokken.Repo.Migrations.AlterOrganizers do
+  use Ecto.Migration
+
+  def change do
+    alter table(:organizers, primary_key: false) do
+      add :first_name, :string, null: false
+      add :last_name, :string, null: false
```

Comment:
We are running in production, meaning the table organizers already has data, which we cannot erase. This migration will fail as the new collumns will always default to null. You must specify a default value for it to work (or make it nullable, but I don't think it is a good idea)